packages/blockchain-extension/extension/commands/deleteNodeCommand.ts

```diff
@@ -51,9 +51,9 @@ export async function deleteNode(nodeTreeItem: NodeTreeItem | FabricNode, hideNo
 
             let chosenEnvironment: IBlockchainQuickPickItem<FabricEnvironmentRegistryEntry>;
             if (hideNode) {
-                chosenEnvironment = await UserInputUtil.showFabricEnvironmentQuickPickBox('Choose an OpsTool environment to hide a node from', false, true, false, EnvironmentType.OPSTOOLSENV, false, true) as IBlockchainQuickPickItem<FabricEnvironmentRegistryEntry>;
+                chosenEnvironment = await UserInputUtil.showFabricEnvironmentQuickPickBox('Choose an OpsTool environment to hide a node from', false, true, false, IncludeEnvironmentOptions.OPSTOOLSENV, false, true) as IBlockchainQuickPickItem<FabricEnvironmentRegistryEntry>;
```

Comment:
Should we change this to the prompt to `OpsTools` instead of `OpsTool` to make it consistent?